src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlInstanceDatabase.md

```diff
@@ -0,0 +1,213 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+online version:
```

Comment:
The ci failure is due to this field missing a value.  Per the ValidateHelpIssues.csv, the value shoudl be: https://docs.microsoft.com/en-us/powershell/module/azurerm.sql/get-azurermsqlinstancedatabase

Reply:
Thanks, just sent the fix.